core/src/main/java/io/questdb/griffin/RecordToRowCopierUtils.java

```diff
@@ -756,7 +764,17 @@ public static RecordToRowCopier generateCopier(
                             break;
                     }
                     break;
+                case ColumnType.UUID:
```

Comment:
this is a top-level type `from` the to type is assumed here is `uuid`, which is very brave. I would like to see a tonne of tests here for attempts to copy to other types.

I guess we should allow `String` as target type?

Reply:
I don't get the first part: this is executed when the top-level type is UUID. what is brave about it? 

I added a code allowing `String` as a target type.

Reply:
with the additional switch below the code looks less brave and good! :)